server/save.js

```diff
@@ -8,6 +8,7 @@ var filetype = require('../content/src/filetype');
 exports.handleSave = function(req, res, app) {
   var data = utils.param(req, 'data');
   var meta = utils.param(req, 'meta');
+  var thumbnail = utils.param(req, 'thumbnail');
```

Comment:
No. `handleSave` will always involve a file, there is no way to save a thumbnail for a directory. We can certainly support that too. But we need to discuss on how the client side sets the thumbnail for a folder.